Testing/Dashboard/Summary/Utils.cs

```diff
@@ -0,0 +1,118 @@
+﻿/// <author>Sairoop Bodepudi</author>
+/// <created>19/11/2021</created>
+/// <summary>
+///		This is the unit testing utilities 
+///		file which would be used while testing
+///		the summarizer functions.
+/// </summary>
+
+using Content;
+using System;
+using System.Collections.Generic;
+
```

Comment:
Can we convert the blocks of codes inside if-else, to some helper functions to avoid high cyclomaticity